RELEASE-NOTES.txt

```diff
@@ -2,6 +2,10 @@
 
 16.8
 -----
+* [*] Block Editor: Fixed block mover title wording for better clarity from 'Move block position' to 'Change block position'. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3049]
+* [**] Block Editor: Make inserter long-press options "add to beginning" and "add to end" always available. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3074]
+* [**] Block Editor: Add support for setting Cover block focal point. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3028]
+* [*] Block Editor: Fix crash when Column block width attribute was empty. [https://github.com/WordPress/gutenberg/pull/29015]
```

Comment:
Same as previous comment, this change was included in the 1.46.1 beta fix so it should better go under 16.7 section.

Reply:
Thank you for the heads up @fluiddot 🙇 
Issue fixed with https://github.com/wordpress-mobile/WordPress-Android/pull/14081/commits/e7298e35489509ce817bb859e9ba572689d0d806